plugins/modules/foreman_location.py

```diff
@@ -26,7 +26,7 @@
 author:
   - "Matthias M Dellweg (@mdellweg) ATIX AG"
 requirements:
-  - "nailgun >= 0.28.0"
+  - "apypie >= 0.0.2"
```

Comment:
I didn't think we are adding explicit version dependency here?

Reply:
Oh, yes. This line was older...